contrib/init/systemd/README.md

```diff
@@ -3,20 +3,37 @@
 > NOTE: These instructions are valid for Ubuntu versions using `systemd` init
 > system.
 
-Run the following commands:
+## TL;DR
+1. Get the latest [release](https://github.com/Azure/azurefile-dockervolumedriver/releases)
+2. Put the binary into `/usr/bin/azurefile-dockervolumedriver`
+3. Get the .default and .service files from the .tar.gz file and deploy them
+4. Reload systemd
+
+## In-depth walkthrough
+To ssh onto machines created with the `docker-machine` commands, you'll probably need to ssh via `docker-machine ssh [machinename]` to perform the following.
 
 0. `sudo -s`
-0. Download the binary from “Releases” tab of the repo to `/opt/bin/`: `wget -qO/usr/bin/azurefile-dockervolumedriver [url]`
-0. Make it executable: `chmod +x /usr/bin/azurefile-dockervolumedriver`
+0. Download the tar.gz from "Releases" tab of the repo
+    + Currently this would be `wget https://github.com/Azure/azurefile-dockervolumedriver/archive/0.2.1.tar.gz` 
+0. Extract and copy the extracted files to `/usr/bin/azurefile-dockervolumedriver`: 
```

Comment:
Sorry, left the remnant in - should be changed to "Extract the files so we can get the relevant config files we need later"